lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -23,6 +23,8 @@ def self.provision_execute(_params = {}, _secrets = {}, context = {})
           return BuiltinRunnner.error!({}, :cause => "Unable to find MiqReqeustTask id: [#{object_id}]")                        if miq_request_task.nil?
           return BuiltinRunnner.error!({}, :cause => "Calling provision_execute on non-provisioning request: [#{object_type}]") unless miq_request_task.class < ::MiqProvision
 
+          new_options = miq_request_task.options.merge(context.input.symbolize_keys)
+          miq_request_task.update!(:options => new_options)
```

Comment:
@kbrock could you use `miq_request_task.options.merge!` then `miq_request_task.save!` to save creating a dup new object ?

Reply:
thnx.

tests still pass when `options_will_change!` was not used, but feels like it should be necessary.